.travis.yml

```diff
@@ -16,40 +16,42 @@ env:
             "5/U+SFc0p1n07i9Hy091YkpLDXxOZviczNYW/juQqQVqqSjwgWoVhkkfy7vJrc0+NwSf8pVD7Lq8WdUhAygbXAN443/R4wATbWi87oLwXrOLLpwWBEDnRzhPxANMq6ADigXm8rjZJnkEDmB2gs8yTOXDVxgysGl1a4+3nXezeITolcaaTgh5x7XslwmFEwhnZxLxh/TkpyEGEtvi6tEB3mCEJzcFQ0DS3InEXp8qJRkX+QH8zJc9cUO9PPJ3BSHzm8MvdXG+CAcaA9UJwonBRsNqDlrQIdZLEMxTzNMnfVOMbam9zMLaeFc0RjcVALKKhS3UJ5xU6CnwBToz9LWTCvi6hs6Ijooj4gYCLTijKgZAheU0/BlNqc6UcL25b9OSMLr1VvFoWxsxMi2tPAj0TnaR+zI03UutGNgbbf659HpyOgi7b/t460OhNnd2HM4xBYbevYqt3t7cQ/43cacCocaC92Y0ZNTEIsJduVt3Un/FPe8Fbmcm/iXCwx0ZI95c0PmAbkz1PipVtVPGPqYMLQi5WLpBAcYUINxREQtoukh6k55pbaHWBNKfkradRQZMRR7/B0xBkJVZwwsxJjg6dbq9HcQQ/t5nZzlCcagn3m1awjIqx7LiyzYF3AvDWL2itpEseA5wIPFadGHPBlt/69lvG1qJ+LnTjzFXSPY4j1w="
 
 language: python
-# The travis Python version is unrelated to the version we build and test
-# with.  This is set with the MB_PYTHON_VERSION variable.
-python: 3.5
-sudo: required
-dist: trusty
+python: 3.6
+os: linux
+dist: bionic
 services: docker
 
-matrix:
+jobs:
   exclude:
-    # Exclude the default (TravisCI provided) Python 3.5 build
-    - python: 3.5
+    - python: 3.6
   include:
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.6
         - TEST_DEPENDS="numpy==1.10.1"
+      python: 3.6
```

Comment:
I don't think we need these lines - at least, the latest astropy does not have them?

https://github.com/MacPython/astropy-wheels/blob/master/.travis.yml

(But let's get the main stumbling block fixed first)

Reply:
I think we should remove these -  it is missing on the Python 3.8 entry for 64bit (i686) which was the only Linux target with worked on this build:

https://travis-ci.org/github/biopython/biopython-wheels/jobs/690700231